org.eclipse.jdt.core.manipulation/core extension/org/eclipse/jdt/internal/corext/callhierarchy/CallHierarchyFilterOptions.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.jdt.internal.corext.callhierarchy;
+
+/**
+ * These are the filter Options for the Call Hierarchy
+ */
+public enum CallHierarchyFilterOptions {
+	SHOW_ALL_CODE("PREF_SHOW_ALL_CODE",  "Show All Code"), //$NON-NLS-1$ //$NON-NLS-2$
+	HIDE_TEST_CODE("PREF_HIDE_TEST_CODE", "Hide Test Code"), //$NON-NLS-1$ //$NON-NLS-2$
+	SHOW_TEST_CODE_ONLY("PREF_SHOW_TEST_CODE_ONLY", "Test Code only"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
You should use the proper CallHierarchyMessages.FiltersDialog_ShowAllCode variables to return the labels.

Reply:
done